js_modules/dagit/packages/core/src/asset-graph/AssetNode.tsx

```diff
@@ -31,7 +31,8 @@ export const AssetNode: React.FC<{
   padded?: boolean;
   inAssetCatalog?: boolean;
 }> = React.memo(({definition, selected, liveData, inAssetCatalog, padded = true}) => {
-  const stepKey = definition.opName || '';
+  const firstOp = definition.opNames.length ? definition.opNames[0] : null;
+  const stepKey = firstOp || '';
```

Comment:
Hmm I'm not sure where this was being used, but i think we want to pass all the opNames anywhere we were passing stepKey

Reply:
This is for highlighting a step on the Run page filtered logs. I don't think we currently support filtering to more than one step.

Reply:
Added a comment here.